pages/use-case/human-resource.vue

```diff
@@ -53,18 +57,22 @@ export default {
     SeamlessIntegration,
     BlogCard,
   },
-  async asyncData({ $content }) {
-    let articles = await $content('blog')
-      .where({
-        published: true,
-        featured: false,
-      })
-      .search('human resource')
-      .sortBy('createdAt', 'desc')
-      .fetch()
+  async asyncData() {
+    const {data: {data}} = await axios.get(
+      `${process.env.strapiUrl}/api/blogs?filters[$or][0][title][$contains]=HR&filters[$or][1][title][$containsi]=human resource&sort=publishedAt:desc&populate=*`
+    )
+
+    let articles = data.map((item) => {
+      return {
+        id: item.id,
+        ...item.attributes,
+        coverImg: item.attributes.coverImg.data.attributes.url,
+        readingStats: readingTime(item.attributes.body),
+      }
+    })
 
-    const randIndex = Math.floor(Math.random() * (articles.length - 3))
-    articles = articles.slice(randIndex, randIndex + 3)
+    const randIndex = Math.floor(Math.random() * (articles.length - 3));
+    articles = articles?.slice(randIndex, randIndex + 3);
```

Comment:
I see this is being used at multiple places. It's good idea to create a component called as `<related-blogs :slug="slug" count="3" />` which takes care of this thing in one component and renders it.